setup.py

```diff
@@ -23,287 +23,6 @@
 
 import setuptools
 
-HAS_DIST_INFO_CMD = False
```

Comment:
As far as I remember, all these were to assure things can be installed on older setuptools. If we're removing it, we need to document this somewhere that we don't support old versions of whatever. It's hard to tell what this will break TBH.

Reply:
Code should never be added without tests that test for regressions. That is why we ended with a magic setup.py that does so much magic that it breaks installation.